docs/more-resources.md

```diff
@@ -16,7 +16,7 @@ There’s always more to learn: developer workflows, shipping to app stores, int
 
 ## Dive deep
 
-- [React’s Documentation](https://reactjs.org/docs/hello-world.html)
+- [React’s Documentation](https://react.dev/reference/react)
```

Comment:
```suggestion
- [React’s Documentation](https://react.dev/learn)
```

It might be a bit better to point to the quick start guide instead of API reference (for hooks, in case of this link).